net/zmap/Portfile

```diff
@@ -15,23 +15,22 @@ long_description    designed for Internet-wide network surveys capable of \
                     scanning the entire IPv4 address space in 5 minutes from \
                     a 10GbE connection.
 homepage            https://zmap.io/
+github.tarball_from archive
 
-checksums           rmd160  4b30c0f33c5e4503a89dcb37808c50075373c4f4 \
-                    sha256  61787bd59cb23183a8b5424a148b95812d8254d6249c2d4efb31cbebff9ca383 \
-                    size    154851
+checksums           rmd160  9ef9b3293de156a76e9e13652cfc46b4321c9b85 \
+                    sha256  0e256d3b62c4ec2f51367c6036aefd99eef6ce6b0b6bcda858223c5a29fda77a \
+                    size    156242
 
 depends_build-append port:byacc \
                     port:flex \
                     port:gengetopt \
                     port:pkgconfig \
-                    port:cmake
+                    port:json-c
```

Comment:
you're not linking with this?

Reply:
Ah, this is probably due to attempting to fix an error in the build bots related to json not being found in a previous effort.

I had previously attempted to break out json-c into a variant, similar to how this Portfile submission was originally: https://trac.macports.org/attachment/ticket/42895/Portfile

Regardless, it's probably better to have port:json-c under depends_lib-append instead of depends_build-append, whoops!

Reply:
well, please check and verify where it is needed

Reply:
@reneeotten thanks for the review! The most recent commit should reflect the appropriate changes.